.github/workflows/aws_compute_ec2_single_region_golden.yml

```diff
@@ -25,15 +19,29 @@ env:
 
     AWS_PROFILE: infex
     AWS_REGION: eu-west-2
-    TF_PATH: ${{ github.workspace }}/aws/compute/ec2-single-region/terraform
-    GOLDEN_PATH: ${{ github.workspace }}/aws/compute/ec2-single-region/test/golden
+    S3_BACKEND_BUCKET: tests-ra-aws-rosa-hcp-tf-state-eu-central-1
+    S3_BUCKET_REGION: eu-central-1
+    S3_BUCKET_KEY: golden.tfstate
 
-    MODULE_DIR: ./aws/ec2/terraform/
+    MODULE_DIR: ./aws/compute/ec2-single-region/terraform/
 
 
 jobs:
+    triage:
+        runs-on: ubuntu-latest
+        outputs:
+            should_skip: ${{ steps.skip_check.outputs.should_skip }}
+        steps:
+            - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

Comment:
not sure how hard you're set on this.
In theory you can skip the checkout step and it would simply default to the main branch for the action use.

Reply:
Maybe just to note as side info.

Reply:
nope, nothing is required.